reports/update-website.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+Rscript data-raw/update.R
```

Comment:
this will update the forecast date. Is this what you want to do here?

Reply:
ah - no. not intentionally. At least I don't intend to do anything with the forecast date anywhere. 
But I guess it doesn't do any harm? This script runs twice - on Sunday (when the new data comes in) and on Monday (after the submission). At that point on Monday all Rt scripts are already done.

Reply:
unless we need to rerun them...... why are we updating data here anyway?

Reply:
Hm I guess you're right and we can just delete that line from the script - data should be up to date anyway

Reply:
I've deleted it now :)